spec/models/spree/adjustment_spec.rb

```diff
@@ -61,38 +61,43 @@ module Spree
       describe "Shipment adjustments" do
         let(:shipping_method) { create(:shipping_method_with, :flat_rate) }
         let(:shipment)        { create(:shipment_with, :shipping_method, shipping_method: shipping_method) }
-        let!(:order)          { create(:order, distributor: hub, shipments: [shipment]) }
+        let(:order)           { create(:order, distributor: hub) }
         let(:hub)             { create(:distributor_enterprise, charges_sales_tax: true) }
-        let!(:line_item)      { create(:line_item, order: order) }
+        let(:line_item)       { create(:line_item, order: order) }
         let(:adjustment)      { order.adjustments(:reload).shipping.first }
 
-        it "has a shipping charge of $50" do
-          adjustment.amount.should == 50
+        describe "the shipping charge" do
+          it "is the adjustment amount" do
+            order.shipments = [shipment]
+
+            adjustment.amount.should == 50
+          end
         end
 
         describe "when tax on shipping is disabled" do
+          before { Config.shipment_inc_vat = false }
           it "records 0% tax on shipment adjustments" do
-            Config.shipment_inc_vat = false
             Config.shipping_tax_rate = 0
+            order.shipments = [shipment]
 
             adjustment.included_tax.should == 0
           end
 
           it "records 0% tax on shipments when a rate is set but shipment_inc_vat is false" do
-            Config.shipment_inc_vat = false
             Config.shipping_tax_rate = 0.25
+            order.shipments = [shipment]
 
             adjustment.included_tax.should == 0
           end
         end
 
         describe "when tax on shipping is enabled" do
-          before do
-            Config.shipment_inc_vat = true
-            Config.shipping_tax_rate = 0.25
-          end
+        before { Config.shipment_inc_vat = true }
```

Comment:
Wrong alignment :warning:

Reply:
Mmm Rubocop is not complaining?

Reply:
rubocup is not running here, I dont know why. But I get it locally. Same as above, I'll get this sorted on a PR after this, 2812 and 2816 are merged.

Reply:
here: #2856